OgcApi.SqlServer.Tests/SqlServerFacts.cs

```diff
@@ -578,5 +580,34 @@ public void DeleteNotExistingFeature()
             var provider = TestProviders.GetDefaultProvider();
             Assert.Throws<ArgumentException>(() => provider.DeleteFeature("PolygonsForInsert", "200"));
         }
+
+        [Fact]
+        public async void GetTile()
+        {
+            var tile = await TestProviders.GetDefaultProvider().GetTileAsync("Polygons", 1, 1, 1);
+            Assert.NotNull(tile);
+        }
+
+        [Fact]
+        public void GetTileUnknownCollection()
+        {
+            Assert.ThrowsAsync<ArgumentException>(() => TestProviders.GetDefaultProvider().GetTileAsync("test", 1, 1, 1));
+        }
+
+        [Fact]
+        public async void GetTileEmptyTile()
+        {
+            var rawTile = await TestProviders.GetDefaultProvider().GetTileAsync("LineStrings", 8, 1, 250);
+            using var memoryStream = new MemoryStream(rawTile);
+            memoryStream.Seek(0, SeekOrigin.Begin);
+            var tile = new MapboxTileReader().Read(memoryStream, new NetTopologySuite.IO.VectorTiles.Tiles.Tile(8,1,250));
+            Assert.True(tile.IsEmpty);
+        }
+
```

Comment:
Add a GetLimits test for the correct collection name in which you check that the result matches expectations.